autosklearn/metrics/__init__.py

```diff
@@ -93,12 +93,12 @@ def __call__(
             raise ValueError(type_true)
 
         if sample_weight is not None:
-            return self._sign * self._score_func(y_true, y_pred,
-                                                 sample_weight=sample_weight,
-                                                 **self._kwargs)
+            return self._score_func(y_true, y_pred,
```

Comment:
As discussed offline, CV grid search expects the user to provide a scoring function, whose return value valuer would be returned without any modification (so if one provides mse, the return value is mse as it is, without negative value). This produces something that does not make sense as scikit learn maximizes.

The user is expected to provide negative mean square valure as a function so that this  grid search makes sense (as explained [here](https://scikit-learn.org/stable/modules/model_evaluation.html))

So I have reverted to have the sign. The use the sign to correct the calculate_score function (where I also added this clarification). That is, calculate score returns the pure value from the scoring function.